Documentation/gettingstarted/encryption.rst

```diff
@@ -12,9 +12,9 @@ Transparent Encryption
 
 This guide explains how to configure Cilium to use IPsec based transparent
 encryption using Kubernetes secrets to distribute the IPsec keys. After this
-configuration is complete all traffic between Cilium-managed endpoints, as well
-as Cilium managed host traffic, will be encrypted using IPsec. This guide uses
-Kubernetes secrets to distribute keys. Alternatively, keys may be manually
+configuration is complete all traffic between Cilium-managed endpoints (pod-to-pod),
+as well as Cilium managed host traffic (host-to-host), will be encrypted using IPsec.
+This guide uses Kubernetes secrets to distribute keys. Alternatively, keys may be manually
```

Comment:
I think this change is a bit misleading. Pods can run in host network namespace, in which case pod-to-pod is actually Cilium-managed *host* traffic. The sentence was also saying that hostns-to-namespaced-pod would be encrypted, whereas the the new version may give the impression this isn't a covered case.

Reply:
Other than the hyphen, the content of this paragraph has not changed, so I'd avoid reformatting it -- Especially because the second to last line is now longer than the others. @Andrey9kin you can address this if Paul provides other feedback to address, but otherwise I don't mind much if we keep the current version.